packages/@ama-sdk/core/package.json

```diff
@@ -36,6 +36,17 @@
       "node": "./dist/cjs/clients/*.js",
       "require": "./dist/cjs/clients/*.js"
     },
+    "./plugins/mock-intercept/angular": {
```

Comment:
Is there a reason for declaring this separately, as it is handled by the declaration of `./plugins/*` just below ?

Reply:
Yes, we can not export the angular plugin in the `index.ts` (targeted by the declaration below) because we do not everybody using the mock interceptor to have a dependency on Angular :S

Reply:
ah, this is nice :)